gtsam/hybrid/tests/Switching.h

```diff
@@ -196,7 +195,7 @@ struct Switching {
   }
 
   /**
-   * @brief Add "mode chain" to HybridNonlinearFactorGraph from M(0) to M(K-2).
+   * @brief Add "mode chain" to HybridNonlinearFactorGraph from M(0) to M(K-1).
```

Comment:
The docstring further corroborates the fact that we should have `K` modes above.

Reply:
K variables, K-1 modes, no?

Reply:
You're saying K+1 variables in some places and K in others. Which is why I am seeing inconsistency.

Reply:
I'll definitely check and fix any issues I see. K variables, K-1 modes is the truth.